source/features/copy-file.tsx

```diff
@@ -3,28 +3,49 @@ import select from 'select-dom';
 import copyToClipboard from 'copy-text-to-clipboard';
 import features from '../libs/features';
 
-function handleClick({currentTarget: button}: React.MouseEvent<HTMLButtonElement>): void {
-	const file = button.closest('.Box');
+function handleClickFactory(markdown?: Element): ({currentTarget: button}: React.MouseEvent<HTMLButtonElement>) => void {
+	return function ({currentTarget: button}: React.MouseEvent<HTMLButtonElement>): void {
+		if (features.isMarkDown()) {
```

Comment:
You don't need this split; you can update the selector in `.blob-code-inner` on line 21 and the rest should work without copying the whole block

Reply:
markdown in fact has blob-code elements containing several lines inside, to retain spacing different operations have to be made. Also two selectors are required.

Reply:
Two selectors yes, but not two whole almost-identical pieces of code. At most you need:

```js
const content = select.all(isMarkDown() ? '.blame-hunk .blob-code' : '.blob-code-inner', file!) 
```

Reply:
I refractored everything out it.

Reply:
I meant this 696b660

Reply:
My bad, I meant https://github.com/sindresorhus/refined-github/pull/1974/commits/46464c8dcdaa9ba36d20848e78900edc6cc8854a

Reply:
I understood you correctly and refractored that, that split is required for spacing. In it's current state it will actually also miss lines of code. With your permission I will force push my refractors.

Reply:
Can you point out which lines of code are missed exactly when clicking on Copy in https://github.com/sindresorhus/refined-github/blob/master/readme.md?

Reply:
so apparently none, it used to be the case for contributing.